packages/neuron-ui/src/widgets/Button/button.module.scss

```diff
@@ -58,7 +58,8 @@
     vertical-align: sub;
     margin-left: 4px;
     & > path {
-      fill: var(--primary-text-color);
+      fill: white;
```

Comment:
Do you check other components that use `Button`?

Reply:
yes. It adapts svg to dark theme